trademarks.md

```diff
@@ -0,0 +1,144 @@
+Sections that are completed should be considered to be accepted and active.**
+
+
+## Introduction
+
+This document outlines the policy of the Project Jupyter ("Jupyter") regarding the use of its trademarks.
+Any use of any Project Jupyter trademark must be in accordance with this policy.
+
+"Jupyter" is a trademark of the NumFOCUS foundation, of which Project Jupyter is a part.
+
+The Jupyter logos (in several variants) are trademarks of NumFOCUS as well.
+
+Registration of the trademarks is pending.
+
+Derivative word marks referring to Jupyter projects or events, such as "JupyterDays," "JupyterLab," "JupyterHub," "Jupyter Notebook" are also covered by this policy.
+
+All trademarks, even those that apply to open source software, must be used according to certain legal requirements. If these requirements are not met, the trademark may be endangered or lost. One of these requirements is for the trademark owner (in this case, NumFOCUS) to maintain standards for using its trademarks, and to enforce acceptable use of the trademarks by taking action against parties that violate those standards.
+
+Trademark law is mainly a way to protect the public, rather than the trademark holder.
+This means that uses of trademarks that confuse consumers are not permitted under law.
+ In our case, "consumers" would include our developer and user community,
+ or anyone else who might be likely to use Jupyter services, software, or applications.
+ As the owner of the trademark, we must be sure the mark is used properly, so the community is not confused.
+ That is what we mean when we say that an unpoliced trademark may be endangered or lost.
+ When the trademark no longer represents a certain level of quality to the community,
+ or no longer indicates that we are the source of the products that bear the trademark,  
+ the trademark loses its value.
+
+Underlying Project Jupyter's trademark policy is a set of guidelines for what is -- and is not -- acceptable use of our trademarks, specifically the word mark "Jupyter", the Jupyter logos, and variations of those marks.
+This policy describes the uses generally approved by Jupyter and NumFOCUS for its trademarks.
+However, if you violate this policy, or otherwise take actions that may compromise the reputation or trademarks of Jupyter, or expose NumFOCUS to liability, NumFOCUS may require you to cease all use of any Jupyter trademark, regardless of the uses allowed in this policy.
+
+
+## General Goals
+
+In general, we want the word mark "Jupyter" and the Jupyter logos to be used with minimal restriction to refer to Jupyter events, software, or services.
+
+We do not want these trademarks to be used:
+
+- to refer to any other events, software, or services
+- in a way that is misleading or may imply association of third-party events, software, or services with Project Jupyter
+- in ways that confuse the community as to whether the Project Jupyter is open source and free to use
```

Comment:
... whether the software created by Project Jupyter is ... ("software" missing, and we should be specific here since the terms about open source apply only to code, not events).